README.md

```diff
@@ -47,11 +47,15 @@ zAppBuild supports a number of build scenarios:
 * **Merge Build** - Build only changed programs which will be merged back into the mainBuildBranch by using a triple-dot git diff. 
 * **Scan Source** - Skip the actual building and only scan source files to store dependency data in collection (migration scenario).
 * **Scan Source + Outputs** - Skip the actual building and only scan source files and existing load modules to dependency data in source and output collection (migration scenario with static linkage scenarios).
+* **Build Preview** - Supplemental build option. Process all phases of the supplied build option, but will not execute the commands. A build report and build result is generated.
```

Comment:
Can you also specify here that the build report status is set to 4, and it is not taken into account for impact build calculation, just like you did in https://github.com/IBM/dbb-zappbuild/blob/4a87211b316c5de1a86071e310f84cef08baddfb/docs/BUILD.md?plain=1#L1214?